checkov/terraform/checks/graph_checks/azure/AzureStorageAccConfig_SAS_expirePolicy.yaml

```diff
@@ -0,0 +1,28 @@
+metadata:
+  id: "CKV2_AZURE_41"
+  name: "Ensure storage account is configured with SAS expiration policy"
+  category: "IAM"
+
+definition:
+  and:
+    - cond_type: "attribute"
+      resource_types: "azurerm_storage_account"
+      attribute: "shared_access_key_enabled"
+      operator: "exists"
+
+    - cond_type: "attribute"
+      resource_types: "azurerm_storage_account"
+      attribute: "shared_access_key_enabled"
+      operator: "equals_ignore_case"
+      value: "true"
```

Comment:
please add an or block, when `shared_access_key_enabled` is set to `false` then it should pass without setting the expiration

Reply:
Okay, will add this.